tools/slintpad/src/lsp.ts

```diff
@@ -56,7 +56,7 @@ function createLanguageClient(
     return client;
 }
 
-export type FileReader = (_url: string) => Promise<string>;
+export type FileReader = (_url: string) => Promise<[string, null | number]>;
```

Comment:
i wonder if it wouldn't be clean to add an named alias for `[string, null|number]` with a comment explaining the meaning of the number.

Reply:
Good idea. I added a patch exporting those types from Rust to WASM/TS and using them wheer appropriate.

The LSP protocol uses `null|integer` for this, so there is no nice named type available there.